ledger/acctupdates.go

```diff
@@ -50,7 +50,9 @@ const (
 	pendingDeltasFlushThreshold = 128
 	// trieRebuildAccountChunkSize defines the number of accounts that would get read at a single chunk
 	// before added to the trie during trie construction
-	trieRebuildAccountChunkSize = 512
+	trieRebuildAccountChunkSize = 16384
```

Comment:
The initial number was aligned with the one in `BalancesPerCatchpointFileChunk`, which seems to be reasonable at the time. However, when rebuilding the trie from scratch, there is no reason to load such a small blocks at a time. If we have enough memory, we should load larger set of account on each iteration. Note that I also added the `trieRebuildCommitFrequency` which decouples the above from the actual Eviction. Together, these would utilize more memory during the building of the trie, and would shorten the time it takes to build it.

Reply:
> there is no reason to load such a small blocks at a time. 
The way I am understanding is that the database operation in not impacted by trieRebuildAccountChunkSize. 
The database is accessed by Next/Scan calls, and it will follow it's on buffering mechanism. 
It will be all the same if trieRebuildAccountChunkSize is set to 1. What am I missing here?

Reply:
We have millions of accounts in the database. I don't know if we have enough memory to load all of them in, so I was processing it in chunks. Do you believe that increasing this from 16K accounts data to 64K accounts data, for instance, would improve the performance ? It's true that we could have used "1" here, but then we're exercising (slightly) larger execution path.
In the future, I'm thinking about batch operation for the Merkle trie, where we would benefit from this implementation.

Reply:
Thank you for the explanation. I am convinced.